x-pack/plugin/esql/qa/testFixtures/src/main/resources/observability.csv-spec

```diff
@@ -0,0 +1,23 @@
+first_no_grouping
+required_capability: fn_first_last
+FROM sample_data
+| STATS event_duration = first_value(event_duration), message = first_value(message)
+;
+
+event_duration:long | message:keyword
+3450233             | Connected to 10.1.0.3
+;
+
+
+first_grouped
+required_capability: fn_first_last
+FROM sample_data
+| STATS event_duration = first_value(event_duration), message = first_value(message) BY hourly = BUCKET(@timestamp, 1 hour) 
```

Comment:
I was hoping the function could be called `first` but this conflicts with nulls ordering keywords: 
https://github.com/elastic/elasticsearch/blob/aaa5ce85a931c64f2f3aa4ace5f8508557a904b2/x-pack/plugin/esql/src/main/antlr/EsqlBaseParser.g4#L232-L234

Please let me know if it is worth to workaround it somehow (may be parsing first/last only in the sorting mode?)
Although it might not be trivial since sort uses expression mode as it is theoretically possible to:
`| SORT first_value(event_duration) nulls first `